scripts/assemble_changelog.py

```diff
@@ -219,8 +219,10 @@ def add_categories_from_text(self, filename, line_offset,
                                        category.name.decode('utf8'))
 
             body_split = category.body.splitlines()
+            re_has_url = re.compile('.*http[s]?://.*')
```

Comment:
I think the exception should only be for a URL that's alone on its line, not for a line containing a URL.
```suggestion
            re_has_url = re.compile(b'\s*[a-z]+://\S$*')
```